### DISCUSSION TITLE
no config.xml file in boinc folder

### DISCUSSION THREAD

**Discussion by leepeter2019** [BOINC/boinc#5498] [Q&A]
This template is for miscellaneous issues not covered by the other issue categories.
i followed the install instruction, but there is no config.xml file in boinc folder.
I saw some samples in boinc homepage, but there should be essential attributes such as 'config', 'data'
do you generate default config.xml file or do i have to write config.xml file?
do you have any example config.xml file?
I switched to 'dev' from 'master', but the result was same.

**Comment by AenBleidd** [comment]
If you are talking about cc_config.xml file, so the easiest way is to go to the BOINC Manager, change any preference there, and this file will be generated for you.
